session6/src/HomeworkS6/HomeworkS6.java

```diff
@@ -0,0 +1,28 @@
+package HomeworkS6;
+
+public class HomeworkS6 {
+    public static void main(String[] args) {
+
+        //Ex:1
+        String stringOne = "OpenAi";
+        String stringTwo = "OpenAi";
+        //System.out.println(stringOne == stringTwo);
+
+        //Ex:2
+        int firstNumber = 26;
+        int secondNumber = 27;
+        //System.out.println(firstNumber == secondNumber);
+        String firstString = "26";
+        String secoundString = "26";
+        //System.out.println(firstString.equals(secondString));
+
+       //Ex:3
```

Comment:
like previously mentioned, move this exercise in a different method and/or class.